### DISCUSSION TITLE
Extending timestream functionalities

### DISCUSSION THREAD

**Discussion by thomasniebler** [aws/aws-sdk-pandas#1337] [Ideas]
In our project, we use timestream as the central database for time series data. We manage several databases and tables inside there. When querying timestream in Sagemaker studio, we'd like to look up some tables without invoking boto3, just using awswrangler.

Would it be of use to include some simple functions into the awswrangler.timestream module like list_dbs() or list_tables(dbname) or is that something that's explicitly not wanted? I'd be happy to help out here.

**Comment by kukushking** [comment]
IMO yes it would definitely be of use - since boto3 is a client library sometimes it's calls are quite verbose, and wrangler could definitely help save some LOC here. In fact we have similar convenience functions e.g. S3 `list_buckets` or Glue Data Catalog's list `databases` that don't return pandas data frames as most of the focus of the library is.

**Reply by thomasniebler** [reply]
Great, then let me draft those convenience functions that ease that lookup a little bit. I'll design them similar to those functions that you mentioned, or would you rather have them return dataframes?
